bindings/zeebe/command/create_instance.go

```diff
@@ -11,21 +11,20 @@ import (
 	"errors"
 	"fmt"
 
-	"github.com/zeebe-io/zeebe/clients/go/pkg/commands"
-
+	"github.com/camunda-cloud/zeebe/clients/go/pkg/commands"
 	"github.com/dapr/components-contrib/bindings"
 )
 
 var (
-	ErrAmbiguousCreationVars = errors.New("either 'bpmnProcessId' or 'workflowKey' must be passed, not both at the same time")
-	ErrMissingCreationVars   = errors.New("either 'bpmnProcessId' or 'workflowKey' must be passed")
+	ErrAmbiguousCreationVars = errors.New("either 'bpmnProcessId' or 'processDefinitionKey' must be passed, not both at the same time")
+	ErrMissingCreationVars   = errors.New("either 'bpmnProcessId' or 'processDefinitionKey' must be passed")
 )
 
 type createInstancePayload struct {
-	BpmnProcessID string      `json:"bpmnProcessId"`
-	WorkflowKey   *int64      `json:"workflowKey"`
-	Version       *int32      `json:"version"`
-	Variables     interface{} `json:"variables"`
+	BpmnProcessID        string      `json:"bpmnProcessId"`
+	ProcessDefinitionKey *int64      `json:"processDefinitionKey"`
```

Comment:
same comment about backwards compatibility.

Reply:
Looking at our documentation, this component is not in GA. So, it can be a breaking change. This change is OK.

https://docs.dapr.io/operations/components/certification-lifecycle/#alpha
https://docs.dapr.io/operations/components/certification-lifecycle/#beta
https://docs.dapr.io/reference/components-reference/supported-bindings/#zeebe-camunda